server/src/internalClusterTest/java/org/elasticsearch/search/query/SearchQueryIT.java

```diff
@@ -1806,7 +1806,7 @@ public void testWildcardQueryNormalizationOnTextField() {
          {
              WildcardQueryBuilder wildCardQuery = wildcardQuery("field1", "Bb*");
              SearchResponse searchResponse = client().prepareSearch().setQuery(wildCardQuery).get();
-             assertHitCount(searchResponse, 1L);
+             assertHitCount(searchResponse, 0L);
 
              wildCardQuery = wildcardQuery("field1", "bb*");
              searchResponse = client().prepareSearch().setQuery(wildCardQuery).get();
```

Comment:
Maybe add a test where the search string is mixed case but set `wildCardQuery.caseInsensitive(true)`